nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-authorizer/src/main/java/org/apache/nifi/authorization/AuthorizerFactoryBean.java

```diff
@@ -315,8 +315,17 @@ private Authorizer createAuthorizer(final String identifier, final String author
             throw new Exception(String.format("Multiple bundles found for the specified authorizer class '%s', only one is allowed.", authorizerClassName));
         }
 
+        // start with ClassLoad from authorizer's bundle
         final Bundle authorizerBundle = authorizerBundles.get(0);
         ClassLoader authorizerClassLoader = authorizerBundle.getClassLoader();
+        logger.info("Got Authorizer ClassLoader from bundle");
```

Comment:
Do we really need these logs?

Reply:
that log statement should definitely go.  Or be changed to debug.  Or be updated to provide actual context.

Reply:
Thanks for your comments. That log statement was removed and the second more meaningful log statement was given more context.